src/extensions/renderer/base/load-listeners.js

```diff
@@ -1740,6 +1743,8 @@ BRp.load = function(){
             r.data.bgActivePosistion = math.array2point( r.touchData.startPosition );
           }
 
+          cy.emit('dragpan');
```

Comment:
I'm not sure if this emit call is necessary. Based on the code comments, etc. it looked like it 'dragpan' should be emitted here, but I wasn't able to see it fire in my testing.